cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -612,7 +614,11 @@ static DataprocConf create(Map<String, String> properties, @Nullable SSHPublicKe
     String initActions = getString(properties, "initActions");
     boolean runtimeJobManagerEnabled = Boolean.parseBoolean(properties.get(RUNTIME_JOB_MANAGER));
     String autoScalingPolicy = getString(properties, "autoScalingPolicy");
-    int idleTTL = getInt(properties, "idleTTL", 0);
+    int idleTTL = getInt(properties, CLUSTER_IDLE_TTL_MINUTES, CLUSTER_IDLE_TTL_MINUTES_DEFAULT);
+    if (skipDelete) {
+      idleTTL = getInt(properties, CLUSTER_IDLE_TTL_MINUTES, 0);
+    }
```

Comment:
Could you do "if else"? This would make the intent more clear and remove unnecessary double getInt call.

Reply:
done